### PR TITLE
[MM-4644] Upgrade Rate Limiting help text

### PR DESCRIPTION
**Pull request by maxerenberg**
#### Summary
This PR adds a note to the help text of the Rate Limiting setting. It explains that the purpose of rate limiting is to prevent overloading the server from too many requests, either by errors in 3rd party applications or if there's an attack on the system

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-4644

#### Release Note
* Help text for Rate Limiting setting was updated to explain purpose of rate limiting

### CODE REVIEW

**Review by hahmadia — APPROVED** [review]

File: components/admin_console/admin_definition.jsx [hunk 1]
```diff
@@ -1367,7 +1367,8 @@ const AdminDefinition = {
                         label: t('admin.rate.enableLimiterTitle'),
                         label_default: 'Enable Rate Limiting:',
                         help_text: t('admin.rate.enableLimiterDescription'),
-                        help_text_default: 'When true, APIs are throttled at rates specified below.',
+                        help_text_default: 'When true, APIs are throttled at rates specified below.\n \nRate limiting prevents server overload from too many requests. This is useful to prevent 3rd party applications or malicous attacks from impacting your server.',
```
Comment (wiersgallak) [inline components/admin_console/admin_definition.jsx:1370]: @cwarnermm Can you confirm how we stylize the word "3rd party"?  Do we spell out as in "third party" or is it good as a number?

Comment (maxerenberg) [reply components/admin_console/admin_definition.jsx:1370]: In the Features section for the Link Metadata Timeout, we use the spelling "third-party". Would it be better to use this form?

Comment (cwarnermm) [reply components/admin_console/admin_definition.jsx:1370]: @maxerenberg @wiersgallak  We use "third-party" rather than "3rd party" for clarity.

**Comment by amyblais** [comment]
@maxerenberg @mkraft It seems that this PR was merged before (QA / dev / PM) reviews were completed?

**Comment by mkraft** [comment]
> @maxerenberg @mkraft It seems that this PR was merged before (QA / dev / PM) reviews were completed?
> 
> Code review checklist documented here: https://developers.mattermost.com/contribute/getting-started/code-review/.

That's my fault. I didn't properly explain the process. Sorry about that @maxerenberg and @amyblais.

**Pull request merged**